common/ieee754_float/f1.ksy

```diff
@@ -0,0 +1,22 @@
+meta:
+  id: f1
+  title: IEEE 754 Min-float
+  license: Unlicense
+  endian: be
```

Comment:
I don't think this `endian: be` is used in this type.

Reply:
I expect https://github.com/kaitai-io/kaitai_struct/issues/155 be fixed someone, so bit-sized types are expected to be laid out correctly. That's why I have marked all the types as `be` because currently they are `be` in that sense and `le` is not supported at all. Anyway, for 1-byte byte-aligned field it shouldn't make any difference.